adafruit_rockblock.py

```diff
@@ -420,3 +420,43 @@ def system_time(self):
             time_now_unix = iridium_epoch_unix + int(secs_since_epoch)
             return time.localtime(time_now_unix)
         return None
+
+    @property
+    def energy_monitor(self):
+        """Report the current accumulated energy usage estimate in microamp hours.
```

Comment:
Suggest removing "Report" from first line of doc string and just make it say "The current accumulated...".